pkg/kubelet/kubelet_node_status.go

```diff
@@ -90,23 +90,23 @@ func (kl *Kubelet) tryRegisterWithAPIServer(node *v1.Node) bool {
 	}
 
 	if !apierrors.IsAlreadyExists(err) {
-		klog.Errorf("Unable to register node %q with API server: %v", kl.nodeName, err)
+		klog.ErrorS(err, "Unable to register node with API server", "node", kl.nodeName)
```

Comment:
I think we should prefer here to report `node` object instead of `kl.nodeName` as this is error from when operating on `node` object, `kl.nodeName` looks like bad case of hidden side effects.
```suggestion
		klog.ErrorS(err, "Unable to register node with API server", "node", klog.KObj(node))
```